lib/active_model/serializer/adapter/json_api/link.rb

```diff
@@ -3,30 +3,43 @@ class Serializer
     module Adapter
       class JsonApi
         class Link
-          def initialize(serializer)
+          def initialize(serializer, value)
             @object = serializer.object
             @scope = serializer.scope
+            if value.respond_to?(:call)
+              self._value = instance_eval(&value)
+            else
+              self._value = value
```

Comment:
with the `serializable_hash` to `as_json` change in https://github.com/rails-api/active_model_serializers/pull/1406/files#r48702394